app/controllers/topics.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+  newTopic: Ember.computed(function () {
+    return this.store.createRecord('topic');
+  }),
+
+  replaceNewTopic () {
+    this.set('newTopic', this.store.createRecord('topic'));
```

Comment:
Well, probably don't use the attribute then. The combination of these two (`newTopic` and `replaceNewTopic()` doesn't really make sense. To reset `newTopic` with the exact contents of `newTopic` feels like a code smell.